lib/mask_vcf.py

```diff
@@ -48,15 +45,8 @@ def readVCF(vcffile):
         continue;
     # Check each SNP in the VCF file; skip the header lines.
 
-    for site in problematic_sites:
-        if vcflines[i][1] == site[1] and vcflines[i][4] in site[4] and vcflines[i][6] == "PASS":
-            vcflines[i][6] = "FILTER";
-            num_filtered += 1;
-    # Check each SNP in the provided -vcf file. If it matches the current SNP, add the filter string to the
-    # FILTER column.
-
```

Comment:
Would it be possible to just comment this block and lines 36-37 above just in case we want to quickly do the masking again at some point in the future?